### PR TITLE
Bug 1215371 - [Email] Wrap last menu button in span to match building block changes

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1215371

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=aa635e5b7c4672db4a958fdc5032720045078266

File: apps/email/js/cards/cmp/contact_menu.html [hunk 1]
```diff
@@ -1,11 +1,13 @@
 <form class="cmp-contact-menu" role="dialog" data-type="action">
   <header></header>
-   <menu>
-     <button class="cmp-contact-menu-edit" data-l10n-id="message-edit-menu-edit">
+  <menu>
+    <button class="cmp-contact-menu-edit" data-l10n-id="message-edit-menu-edit">
```
Comment (KevinGrandon) [inline apps/email/js/cards/cmp/contact_menu.html]: Previous whitespace seemed wrong here.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=176dbe058b80f9a61ced9e4d72db80f247a8f045

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ea36c770286644962923a09116149f0293ca2a00

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=83918ae90c5d4e1f49d6fc51e154859fa2583dc6

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=41904e05f906167f732ecbda60e00a3d39e0ba76

**Pull request merged**